lib/tests/test_userdb.c

```diff
@@ -73,7 +73,13 @@ Test(user_db, resolve_user_group)
 {
   gint uid;
   gint gid;
-  char str[] = "root:root";
+  char str[] = "root:"
+#ifdef __FreeBSD__
+               // There is no "root" group on FreeBSD: group with gid==0 is named "wheel".
+               "wheel";
+#else
+               "root";
+#endif
 
```

Comment:
hmmm.. is there any chance we can find a group that is available on all our platforms? I hate these kind of ifdefs scattered around in the code.

Reply:
I quite agree, but am not sure this is really reliable.  Checking `getent group` on a FreeBSD and a Debian box, I found only 2 matches:

```
daemon:*:1:
sys:*:3:
```

So I switched to looking for `root:daemon` and expecting to find UID 0 and GID 1.

Reply:
On CentOS 7 and Arch Linux, daemon is GID 2:
```
daemon:x:2:
```

Reply:
`sys` seems to be GID 3 on CentOS, Ubuntu, Debian and Arch too, but I'm not sure we should assert on exact values here.

Reply:
:smile:, looks like _sys_ was introduced with GID 2 back in 1972 with research UNIX V7:
https://github.com/dspinellis/unix-history-repo/blob/Research-V7-Snapshot-Development/etc/group

So for sure, we can't count on anything "portable" here…  We can extract the name and group of the current user and ensure we get the correct UID & GID, but since we need the UID and GID to find a user and group name, I am not sure it makes any sense…

I'll change to use _root:sys_, which should be better than _root:daemon_ here, thanks!